views/index.erb

```diff
@@ -1,6 +1,6 @@
 <!-- node status on /clusters page -->
-<% @clusters.each_with_index do |cluster, index | %>
-    <% if index == @clusters.count - 1 && @clusters.count.odd? %>
+<% CLUSTERS.each_with_index do |cluster, index | %>
+    <% if index == CLUSTERS.count - 1 && CLUSTERS.count.odd? %>
         <div class="col-sm-3"></div>
```

Comment:
Instead of adding an empty div I think you can instead add an offset class to `<div class="col-sm-6 col-xs-12">` https://getbootstrap.com/docs/3.3/css/#grid-offsetting i.e.

```ruby
<div class="col-sm-offset-3 col-sm-6 col-xs-12">
```

Which means you could do something like:

```ruby
<div class="<%= 'col-sm-offset-3' if index == CLUSTERS.count - 1 && CLUSTERS.count.odd? %> col-sm-6 col-xs-12">
```

removing the control flow.

Or you could just get rid of this shift, I'm not sure what it adds. Centering the bottom cluster seems to draw more attention to it over the others. Without the shift it looks like this:

![screen 2018-06-25 at 5 45 42 pm](https://user-images.githubusercontent.com/512333/41877548-b075e362-789f-11e8-896d-5980ea299a31.png)

Reply:
Keep the centering. Alan likes it.